manifests/repo.pp

```diff
@@ -28,10 +32,11 @@
 # Copyright 2014 Werner Dijkerman
 #
 class zabbix::repo (
-  Boolean                                              $manage_repo    = $zabbix::params::manage_repo,
-  Boolean                                              $manage_apt     = $zabbix::params::manage_apt,
-  Variant[String[0],Stdlib::HTTPUrl, Stdlib::HTTPSUrl] $repo_location  = $zabbix::params::repo_location,
-  String[1]                                            $zabbix_version = $zabbix::params::zabbix_version,
+  Boolean                                              $manage_repo                = $zabbix::params::manage_repo,
+  Boolean                                              $manage_apt                 = $zabbix::params::manage_apt,
+  Variant[String[0],Stdlib::HTTPUrl, Stdlib::HTTPSUrl] $repo_location              = $zabbix::params::repo_location,
+  Variant[String[0],Stdlib::HTTPUrl, Stdlib::HTTPSUrl] $unsupported_repo_location  = $zabbix::params::unsupported_repo_location,
```

Comment:
I think we can remove `String[0]` here? Also the `Stdlib::HTTPUrl` type allows HTTPS:
```suggestion
  Stdlib::HTTPUrl $unsupported_repo_location  = $zabbix::params::unsupported_repo_location,
```

Reply:
Please ensure that the `=` of the parameters are still aligned after updating the types.